### DISCUSSION TITLE
Anfängerprobleme

### DISCUSSION THREAD

**Discussion by timi35** [roe-dl/weewx-DWD#53] [Q&A]
Hallo,

ich bin Anfänger bei der Nutzung von weewx bzw. dem DWD Plugin.
Ich bin nach Aneleitung vorgegangen erhalte abe beim Einbinden von:

[Engine]
    
    [[Services]]
        data_services =  user.weatherservices.DWDservice

Folgenden Fehler:
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****      obj = weeutil.weeutil.get_object(svc)(self, config_dict)
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****    File "/usr/share/weewx/user/weatherservices.py", line 1532, in __init__
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****      self._create_poi_thread(location, station, location_dict)
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****    File "/usr/share/weewx/user/weatherservices.py", line 1563, in _create_poi_th>
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****      log_success=weeutil.weeutil.to_bool(station_dict.get('log_success',False)) >
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****  AttributeError: 'DWDservice' object has no attribute 'verbose'
Jul 17 15:43:28 raspberrypi python3[8254]: weewx[8254] CRITICAL __main__:     ****  Exiting.
Jul 17 15:43:28 raspberrypi systemd[1]: weewx.service: Main process exited, code=exited, status=1/FAILURE
Jul 17 15:43:28 raspberrypi systemd[1]: weewx.service: Failed with result 'exit-code'.
Jul 17 15:43:28 raspberrypi systemd[1]: weewx.service: Consumed 2.284s CPU time.

Was habe ich falsch gemacht? Was habe ich evtl. übersehen?
Gibt es irgendwo eine Anleitung für Anfänger? Die bei Github scheint etwas zu "springen", was es für Anfänger nicht gerade einfach macht.

Vielen Dank!
VG Tim

**Comment by roe-dl** [comment]
Bitte `log_success` und `log_failure` in der Konfiguration setzen. Dann sollte der Fehler nicht mehr auftreten.

**Comment by timi35** [comment]
Hallo, ein Stück weiter das hat erstmal für Besserung gesorgt. Wobei ich das bisher deaktiviert hatte, damit der Raspi nicht so viel mitloggen muss. Gibt es eine Möglichkeit beides unter einen Hut zu bekommen?

Nach der Änderung der Konfig in:

    # Stationen
    [[current]]
        [[[10488]]]
            provider = DWD
            model = POI

Werden jetzt Daten geladen. Ich weiß zwar noch nicht whon, aber das Log zeigt es zumindest an.
In Verzeichnis /etc/weewx/skins/Belchertown/dwd kommt es jedenfalls nicht an.

Vielen Dank für die Unterstützung, es kommen sicher noch ein paar Fragen:-)
VG Tim

**Comment by roe-dl** [comment]
> Hallo, ein Stück weiter das hat erstmal für Besserung gesorgt. Wobei ich das bisher deaktiviert hatte, damit der Raspi nicht so viel mitloggen muss. Gibt es eine Möglichkeit beides unter einen Hut zu bekommen?

Man kann beide auf `false` setzen, dann loggt er nicht. Solange noch nicht alles wie gewünscht funktioniert, sollte man es aber anstellen, um Hinweise zu bekommen, was schiefläuft.

Für den anderen Teil der Frage bräuchte ich mehr Informationen, um ihn beantworten zu können.

**Comment by timi35** [comment]
Nachdem ich nun offenbar alles richtig konfiguriert habe, scheint es nun auch mit Log = false zu laufen.
Ich muss sagen, Belchertown mit DWD war bisher am schwierigsten anzupassen. 

VG Tim
